### PR TITLE
Block creative and godmode pvp

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
It gets annoying, I know they should be punished but this makes it easier.

### CODE REVIEW

**Comment by VjGamingUK** [comment]
Neutral.

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java [hunk 1]
```diff
@@ -924,3 +924,32 @@ public void onPlayerLogin(PlayerLoginEvent event)
...
+    public void onEntityDamageByEntity(EntityDamageByEntityEvent event)
+    {
+        if (event.getEntity() instanceof Player)
+        {
+            if (event.getDamager() instanceof Player)
+            {
+                Player player = (Player) event.getDamager();
+                if (player.getGameMode() == GameMode.CREATIVE)
+                {
+                    TFM_Util.playerMsg(player, "NO GM / GOD PVP!", ChatColor.DARK_RED);
```
Comment (Ptatoz) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: I think it should say "Creative (gm) PVP is not allowed"

Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: @Ptatoz @LegendIsAwesomes How about: `ChatColor.GRAY + "Creative PVP is prohibited."`

**Comment by JeromSar** [comment]
@LegendIsAwesomes Pretty sure this doesn't work for Godmode? Essentials integration?

**Comment by Hockeyfan360** [comment]
No, because sometimes, I like to hit players in order to tell which gm they are in.

**Comment by LegendIsAwesomes** [comment]
@Hockeyfan360 why not use / who is (name) that tells you there gamemode and if they're in god

**Comment by Hockeyfan360** [comment]
@LegendIsAwesomes Its less convenient. It is just lazy not to add a check for admin.

**Comment by LegendIsAwesomes** [comment]
https://github.com/LegendIsAwesomes/TotalFreedomMod-1/blob/master/src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java @JeromSar

**Comment by LegendIsAwesomes** [comment]
@JeromSar the above edit.

**Comment by Wild1145** [comment]
What issue does this even resolve?

**Comment by LegendIsAwesomes** [comment]
@Wild1145 Stops creative pvp in more of a effective way, Once a op gets smited they continue, Even after bans they continue so this will stop it completely.

**Comment by Wild1145** [comment]
@LegendIsAwesomes Okay, and what issue number does this PR Link to?

**Comment by LegendIsAwesomes** [comment]
@Wild1145 I don't think this links to a issue, Not sure, I'll check through, But it will solve the creative pvp issue (In game) in a effective and quick way.

**Comment by Wild1145** [comment]
@LegendIsAwesomes If it does not link to an issue you need to create one, all PR's ***MUST*** link to an issue, ideally an approved one.

**Comment by LegendIsAwesomes** [comment]
@Wild1145 My GTFO one didn't have a issue, It was still accepted its not ready to merge

**Comment by Wild1145** [comment]
@LegendIsAwesomes That may be so, however the contributing guidelines require an issue to be associated with every PR...

**Comment by LegendIsAwesomes** [comment]
@Wild1145 I'll create one soon. But @JeromSar my GTFO one didn't have a issue, It was still accepted its not ready to merge

**Comment by Wild1145** [comment]
@LegendIsAwesomes As I say, guidelines require an issue, exceptions are made from time to time, but really you should always have an associating issue.

**Comment by JeromSar** [comment]
Requires associated issue.

**Comment by JeromSar** [comment]
Won't compile.

**Comment by LegendIsAwesomes** [comment]
@JermSar give me some time and I'll fix this, don't have access to my pc ATM.

**Comment by LegendIsAwesomes** [comment]
@JeromSar I deleted my fork of this so I cannot edit without making another PR.

**Comment by JeromSar** [comment]
@

**Comment by LegendIsAwesomes** [comment]
@JeromSar shall I create a new PR?

**Pull request closed without merging**